### PR TITLE
geopandas - for spatial join

### PR DESCRIPTION
**Pull request by alexisbcook**
fix for this issue: https://www.kaggle.com/product-feedback/53008

BUG=118703729

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -554,3 +554,8 @@ ADD patches/tensorboard/notebook.py /opt/conda/lib/python3.6/site-packages/tenso
 
 # Set backend for matplotlib
 ENV MPLBACKEND "agg"
+
+# geospatial learn course
+RUN pip install rtree && \
```
Comment (rosbo) [inline Dockerfile]: Could you move this up to line 494?
https://github.com/Kaggle/docker-python/blob/abf2e82f4c60549ec055c553b524fcbcf1074843/Dockerfile#L494

Comment (alexisbcook) [reply Dockerfile]: done! thanks

File: Dockerfile [hunk 2]
```diff
@@ -554,3 +555,7 @@ ADD patches/tensorboard/notebook.py /opt/conda/lib/python3.6/site-packages/tenso
 
 # Set backend for matplotlib
 ENV MPLBACKEND "agg"
+
+# geospatial learn course
+RUN pip install osmnx && \
```
Comment (rosbo) [inline Dockerfile]: Could you move up these commands too? Thanks

Comment (alexisbcook) [reply Dockerfile]: done!

**Pull request merged**